### PR TITLE
collection branch for all breaking changes

### PR DESCRIPTION
**Pull request by trieloff**
- [x] introduce domain keys filtering #835 
- [x] remove the `generation` filters and parameters #835 
- [x] unify `domain` and `url` parameters #833 
- [x] remove unused queries #832 
- [x] unify `interval`, `offset`, `startdate`, and `enddate` parameters #834

### CODE REVIEW

**Comment by trieloff** [comment]
@meryllblanchet once this has been merged, we need to look at how the Experimentation overlay will still be able to access the data, as the goal would be to turn off the `@v2` endpoint quite quickly.

**Comment by meryllblanchet** [comment]
Hello @trieloff completely missed this one!
Let me loop-in @ramboz for visibility.

File: src/util.js [hunk 1]
```diff
@@ -141,19 +141,21 @@ export function csvify(arr) {
...
-    ':names': ['result', 'meta'],
+    ':names': ['results', 'meta'],
     ':type': 'multi-sheet',
     ':version': 3,
     results: {
       limit: Math.max(requestParams.limit || 1, results.length),
       offset: requestParams.offset || 0,
       total: requestParams.offset || 0 + results.length + (truncated ? 1 : 0),
       data: results,
+      columns: Object.keys(results[0] || {}),
```
Comment (tripodsan) [inline src/util.js:152]:
```suggestion
      columns: Object.keys(results[0] || []),
```

**Comment by mfrisbey** [comment]
LGTM. Merge it!

**Pull request merged**